### PR TITLE
Feat accordion design changes

### PR DESCRIPTION
**Pull request by bruno-sch**
## Proposed changes

Adds design changes to accordion and accordion item component. New variant "card" is added.

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: packages/components/src/components/accordion-item/accordion-item.scss [hunk 1]
```diff
@@ -67,3 +57,23 @@
 		}
 	}
 }
+
+/*
+The variant card can be implicitly set by the surrounding accordion component.
+ */
+.db-accordion[data-variant="card"] {
```
Comment (nmerget) [inline packages/components/src/components/accordion-item/accordion-item.scss:64]: We should move this to `accordion.scss`

File: packages/components/src/components/accordion/accordion.scss [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+	}
+}
+
+/*
+The styling for the spacings between the items is defined in the accordion, where they meet.
+The spacings are not part of the styling of the accordion items themselves.
+ */
 .db-accordion {
-	// No styling for now only JS code for this component
+	&[data-variant="background"] {
```
Comment (nmerget) [inline packages/components/src/components/accordion/accordion.scss:17]: We should add a default as well: ``&:not([data-variant])`

File: packages/components/src/components/accordion/accordion.lite.tsx [hunk 3]
```diff
@@ -126,7 +126,8 @@ export default function DBAccordion(props: DBAccordionProps) {
 		<div
 			ref={ref}
 			id={props.id}
-			class={cls('db-accordion', props.className)}>
+			class={cls('db-accordion', props.className)}
+			data-variant={props.variant ?? 'background'}>
```
Comment (nmerget) [inline packages/components/src/components/accordion/accordion.lite.tsx:130]: We don't need the default, if we add it to css

File: packages/components/src/components/accordion-item/accordion-item.scss [hunk 4]
```diff
@@ -67,3 +57,23 @@
...
+	.db-accordion-item {
+		&::after {
```
Comment (nmerget) [inline packages/components/src/components/accordion-item/accordion-item.scss:66]: Is there any reason for this `::after`? 
Coudn't you add the `%component-border` to the `details`(`db-accordion-item`) element?

File: packages/components/src/components/accordion-item/accordion-item.scss (hunk 1 above)
Comment (bruno-sch) [reply packages/components/src/components/accordion-item/accordion-item.scss]: Done

File: packages/components/src/components/accordion/accordion.scss (hunk 2 above)
Comment (bruno-sch) [reply packages/components/src/components/accordion/accordion.scss]: Done

File: packages/components/src/components/accordion/accordion.lite.tsx (hunk 3 above)
Comment (bruno-sch) [reply packages/components/src/components/accordion/accordion.lite.tsx]: Done

File: packages/components/src/components/accordion/accordion.scss [hunk 5]
```diff
@@ -1,3 +1,53 @@
...
+	}
+}
+
+/*
+The styling for the spacings between the items is defined in the accordion, where they meet.
+The spacings are not part of the styling of the accordion items themselves.
+ */
 .db-accordion {
-	// No styling for now only JS code for this component
+	&:not([data-variant]) {
```
[the remark this answers is not in the document]
Comment (bruno-sch) [reply packages/components/src/components/accordion/accordion.scss:21]: The background variant is actually the default, so no explicit selector is required.

**Review by nmerget — APPROVED** [review]

**Pull request merged**